django/core/validators.py

```diff
@@ -97,7 +97,11 @@ def __init__(self, schemes=None, **kwargs):
             self.schemes = schemes
 
     def __call__(self, value):
-        # Check first if the scheme is valid
+        # Check first if the url is string or contains '://'
+        if not value or '://' not in str(value):
```

Comment:
I changed to `isinstance(value, str)` because it worked properly for an empty string or strings without a scheme.